<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogClass</class>
 <widget class="QDialog" name="dialogClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医院管理系统</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>病人信息查询</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>581</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>258</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>编号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="all_patient">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示全部病人信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="doc_quer_wid">
    <attribute name="title">
     <string>医生信息查询</string>
    </attribute>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>591</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>260</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>编号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="doc_id_qu"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="doc_query">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="all_doc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示全部医生信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="patient_input">
    <attribute name="title">
     <string>病人信息录入</string>
    </attribute>
    <widget class="QPushButton" name="patient_insert">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>470</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>病人姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>病人编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>病人联系方式：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="patient_name">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="patient_id">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="patient_phone">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="doctor_input">
    <attribute name="title">
     <string>医生信息录入</string>
    </attribute>
    <widget class="QPushButton" name="doc_insert">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>440</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>72</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>医生姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>医生编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="doc_id">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="doc_name">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
